pkg/apis/kappctrl/v1alpha1/types.go

```diff
@@ -76,6 +76,7 @@ type AppStatus struct {
 
 type AppStatusFetch struct {
 	Stderr    string      `json:"stderr,omitempty"`
+	Stdout    string      `json:"stdout,omitempty"`
```

Comment:
this might get pretty wordy especially for output vendir produces. i wonder if we need a separate issue to truncate it or something?